packages/local_auth/local_auth_windows/lib/types/auth_messages_windows.dart

```diff
@@ -0,0 +1,191 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:intl/intl.dart';
+import 'package:local_auth_platform_interface/types/auth_messages.dart';
+
+/// Windows side authentication messages.
+///
+/// Provides default values for all messages.
+@immutable
+class WindowsAuthMessages extends AuthMessages {
+  /// Constructs a new instance.
+  const WindowsAuthMessages({
+    this.biometricHint,
+    this.biometricNotRecognized,
+    this.biometricRequiredTitle,
+    this.biometricSuccess,
+    this.cancelButton,
+    this.deviceCredentialsRequiredTitle,
+    this.deviceCredentialsSetupDescription,
+    this.goToSettingsButton,
+    this.goToSettingsDescription,
+    this.signInTitle,
```

Comment:
Are these all actually used? Are any of them?

Reply:
Answering my own question: none of them are, unless I missed something. So this class can be empty for now.

Or not exist, but that's probably more confusing since people would probably look for it and not be able to find it. I would leave it here, but empty (with `args` returning an empty dict) and a comment saying that it's currently unused.

Reply:
Fixed.